Adding BigQuery label length req (#1631)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/bigquery-configs.md

dbt supports the specification of BigQuery labels for the tables and <Term id="view">views</Term> that it creates. These labels can be specified using the `labels` model config.

The `labels` config can be provided in a model config, or in the `dbt_project.yml` file, as shown below.
  

> :::note

Reviewer: ```suggestion
:::info Note
```

dbt supports the specification of BigQuery labels for the tables and <Term id="view">views</Term> that it creates. These labels can be specified using the `labels` model config.

The `labels` config can be provided in a model config, or in the `dbt_project.yml` file, as shown below.
  

> **Note:** Per BigQuery requirements, both key-value pair entries for labels have a maximum length of 63 characters.

Reviewer: Looks good, just adding our note format:

```suggestion
:::note
BigQuery requires that both key-value pair entries for labels have a maximum length of 63 characters.
:::
```
